basics/sourcemaps.md

```diff
@@ -3,6 +3,87 @@ order: -60
 title: Source maps
 ---
 
-!!! Sorry...
-This page is not yet written, please check back soon!
-!!!
+# Client-side debugging with Source Maps
+
+## Sourcemap
+
+Source maps are a way to map a combined/minified file back to the original source files.
+You can access the source, from within a browser, and set a breakpoint.
+
+## Embedding source maps in WebSharper libraries
+
+For WebSharper 4.2+ projects, you can enable including source maps and the required source files in a WebSharper assembly, by adding `"sourceMap": true` to wsconfig.json.
+
+Here is an example of wsconfig.json:
+
+```
+{
+  "project": "site",
+  "outputDir": "wwwroot",
+  "sourceMap": true,
+  "downloadResources": true
+}
+```
+
+For older versions, you can add the `WebSharperSourceMap` property to your project file.
+
+Here is a .csproj / .fsproj sample:
+```
+<PropertyGroup>
+    <WebSharperSourceMap>True</WebSharperSourceMap>
+</PropertyGroup>
+```
+
+WebSharper itself is built with source map embedding, so you can debug into
+WebSharper library code.
+
+Sitelets and single-page application projects are supported, offline sitelets 
+(generating static Html+JS output) is currently not.
```

Comment:
I believe this is being addressed in https://github.com/dotnet-websharper/core/issues/1307

Reply:
I will test it. Thank you!